prow/plank/plank_test.go

```diff
@@ -510,6 +516,7 @@ func TestSyncKubernetesJob(t *testing.T) {
 			expectedPodHasName: true,
 			expectedNumPods:    1,
 			expectedReport:     true,
+			expectedURL:        "NEWPOD/pending",
```

Comment:
Now I see why you need to fake this out. Ideally, we should use a fake kubernetes client and have a stub that returns the pod with a name we want from CREATE.